includes/polling/ports.inc.php

```diff
@@ -472,14 +460,22 @@
             $this_port['ifPhysAddress']              = zeropad($a_a).zeropad($a_b).zeropad($a_c).zeropad($a_d).zeropad($a_e).zeropad($a_f);
         }
 
-        // use HC values if they are available
-        foreach ($hc_mappings as $hc_oid => $if_oid) {
-            if (isset($this_port[$hc_oid]) && $this_port[$hc_oid]) {
-                d_echo("$hc_oid ");
-                $this_port[$if_oid] = $this_port[$hc_oid];
-            } else {
-                d_echo("$if_oid ");
+        $hc_enabled = false;
+        // use HC values if they are available and not disabled
+        if ($config['os'][$device['os']]['no_hc'] != true) {
```

Comment:
Added the option to be able to skip using hc counters. I've helped someone recently where the hc values are utterly broken.